openedx/core/djangoapps/content_libraries/migrations/0006_auto_20210615_1916.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.20 on 2021-06-15 19:16
```

Comment:
This is really minor so don't both fixing it on its own if there's nothing else to fix, but I try to avoid leaving migrations named `auto_....` - it's much more helpful to give them a clear name like `0008_muliple_lti_per_library`